<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Interval</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="sbInterval">
     <property name="toolTip">
      <string>Readout interval requesting the latest datapoint, if not followed.</string>
     </property>
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>40</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Follow</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbFollow">
     <property name="toolTip">
      <string>Follow each data point published by the data logger, otherwise, use the timer to request data points.</string>
     </property>
     <property name="text">
      <string>Follow each data point of the data logger.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data limit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbDataLengthLimit">
     <property name="toolTip">
      <string>Number of data points to keep at least in memory, if the data limit option is activated.</string>
     </property>
     <property name="suffix">
      <string> samples</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFollow</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbInterval</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
